Encounter Me/Services/DeviceValidator.cs

```diff
@@ -2,14 +2,17 @@
 using System.Threading.Tasks;
 using BlazorCurrentDevice;
 using Microsoft.AspNetCore.Components;
+using BlazorCurrentDevice;
 
 public class DeviceValidator
 {
     [Inject] IBlazorCurrentDeviceService BlazorCurrentDeviceService { get; set; }
     public async Task ValidateDevice()
     {
-        bool mobile = await BlazorCurrentDeviceService.Mobile();
-        if (!mobile)
+        string g = await BlazorCurrentDeviceService.Type();
+
+        Console.WriteLine(g);
+        if (g != "Desktop")
         {
             throw new DeviceNotSupportedException();
         }
```

Comment:
I may be wrong, but I thought that desktop is not supported device? This `if` condition states that everything but desktop is not supported?

Reply:
Currently device validator stopped working for me, this was just for testing purposes, this script is not being used anywhere though.

Reply:
Okey